admin/articles.rb

```diff
@@ -249,14 +249,19 @@
 
   member_action :set_page_online_offline do
     article = Goldencobra::Article.find_by_id(params[:id])
-    if article.active
-      article.active = false
-      flash[:notice] = I18n.t('active_admin.articles.member_action.flash.article_offline')
+
+    if article
```

Comment:
It is not a single line with setting flash method and redirecting and to avoid duplication of `redirect_to action: :index`.